src/utils/stripDiacritics.ts

```diff
@@ -114,7 +114,7 @@ export default function stripDiacritics(str: string): string {
   return (
     str
       .normalize('NFD')
-      .replace(/[\u0300-\u036F]/g, '') // Remove combining diacritics
+      .replace(/[\u0300-\u036F\u3099\u309A]/g, '') // Remove combining diacritics
```

Comment:
I'm not an expert when it comes to Japanese diacritics (or regexes for that matter), but it looks to me like this is omitting just two specific characters rather than a range. Am I interpreting that correctly?

Reply:
Hi @ericgio 
Apologies for not replying sooner - I checked the issue page from time to time but didn't think to check the pull request page, and just recently noticed the old notifications in my GitHub inbox. (I'm not very used to contributing to other projects on GitHub...)
You are correct that my change just omits two specific characters.
And sorry for the package.json change, that was me messing around in my fork, I didn't realize you could see those changes also.
Anyway thanks for incorporating the change! I was using a local import of my fork and will now go back to importing the official package. And thanks for all your work on this project!